cryptol-remote-api/python/cryptol/__init__.py

```diff
@@ -16,599 +16,17 @@
 from . import cryptoltypes
 from . import solver
 from .bitvector import BV
+from .commands import *
+from .connection import *
 
 
 __all__ = ['cryptoltypes', 'solver']
```

Comment:
Should this include `commands` and `connection` as well now? (That may require additional imports such as `from . import commands` if so.)

Reply:
I suppose so? I don't know what the philosophy should be behind `__all__` – it already seemed strange to me that the behavior of `from cryptol import *` was to only import `cryptoltypes` and `solver`, and not `connect`, etc.

Reply:
Yah, I was just going off what's already there and skimming this: https://docs.python.org/3/tutorial/modules.html

Probably worth adding them if for no other reason than consistency...?

Reply:
Ah, that link is very helpful - thanks!